src/Documentation/SidebarMenu/SidebarMenu.js

```diff
@@ -66,63 +68,46 @@ export default class SidebarMenu extends React.Component {
 
                     {/* Section Files */}
                     <Collapse data-open={isSectionActive ? 'true' : 'false'}>
-                      {files &&
-                        files.map((fileOrGroup, fileIndex) => {
-                          const file = Array.isArray(fileOrGroup)
-                            ? fileOrGroup[0]
-                            : fileOrGroup
-                          const subgroup = Array.isArray(fileOrGroup)
-                            ? fileOrGroup.slice(1)
-                            : null
-                          const isFileActive = currentFile === file
-                          return (
-                            <Fragment key={`file-${fileIndex}`}>
-                              <div>
-                                <SectionLink
-                                  level={2}
-                                  href={getLinkHref(index, file)}
-                                  onClick={e => onFileSelect(file, index, e)}
-                                  isActive={isFileActive}
-                                >
-                                  {labels[file] || startCase(file.slice(0, -3))}
-                                </SectionLink>
-                              </div>
-
-                              {/* Subgroup files */}
-                              {subgroup && (
-                                <Collapse
-                                  data-flag={'first'}
-                                  data-open={
-                                    Array.isArray(fileOrGroup) &&
-                                    includes(fileOrGroup, currentFile)
-                                      ? 'true'
-                                      : 'false'
-                                  }
-                                >
-                                  {subgroup.map((sub, subIndex) => {
-                                    return (
-                                      <div
-                                        key={`file-${fileIndex}-${subIndex}`}
+                      {files && files.map((file, fileIndex) => {
```

Comment:
let's move into a function that takes `files` and returns s list of `Fragments`. Then we can use it recursively instead of duplicating the code for second and third level. It's hard to maintain, read, scale, etc.